src/constructs/ec2/security-groups.ts

```diff
@@ -47,8 +65,8 @@ export class GuWazuhAccess extends GuSecurityGroup {
       overrideId: true,
       allowAllOutbound: false,
       egresses: [
-        { range: Peer.anyIpv4(), port: Port.tcp(1514), description: "wazuh event logging" },
-        { range: Peer.anyIpv4(), port: Port.tcp(1515), description: "wazuh agent registration" },
+        { range: Peer.anyIpv4(), port: 1514, description: "wazuh event logging" },
+        { range: Peer.anyIpv4(), port: 1515, description: "wazuh agent registration" },
```

Comment:
```suggestion
        { range: Peer.anyIpv4(), port: 1514, description: "Wazuh event logging" },
        { range: Peer.anyIpv4(), port: 1515, description: "Wazuh agent registration" },
```

Since we are defining defaults for the department, the pedant in me would like to see consistent capitalisation in the AWS console. 😅 

TIL: [multi-line suggestions](https://github.blog/changelog/2020-02-26-multi-line-code-suggestions-beta/)